tests/bootstrap.php

```diff
@@ -47,5 +47,6 @@ class_alias("Contao\\$class", $class);
 
 /** @var Composer\Autoload\ClassLoader $loader */
 $loader->addPsr4('Contao\\CoreBundle\\Test\\', __DIR__);
+$loader->addPsr4('Contao\\Fixtures\\', __DIR__.'/Fixtures');
```

Comment:
The namespace is used in https://github.com/contao/core-bundle/pull/76/files#diff-f95982ace6744c19f506f6fb5e0b26b6R11, therefore he had to add this here.

Problem here is, we defined our fixtures with the old autoloader compat to reside in `tests/Fixtures/library` and having the complete namespace of `Contao\\Fixtures`. I guess @dtomasi was confused by this unorthodox directory layout.